### PR TITLE
Feat/new site

### PR DESCRIPTION
**Pull request by hyperlolo**
#1

### CODE REVIEW

File: README.md
Comment (ggiande) [inline README.md:1]: Add README.md again, and populate with meaningful information, please ask manager for more info

**Comment by ggiande** [comment]
Make sure to introduce the following/look into the following:
- [ ] JEST Testing https://jestjs.io/
- [ ] Converting images to .webp  and placing in public/ directory

File: src/main.tsx
Comment (ggiande) [inline src/main.tsx:1]: Make sure to use BrowserRouter  only, do not alias

File: src/modules/header-menu/header.tsx
Comment (ggiande) [inline src/modules/header-menu/header.tsx:1]: Remove unused code

File: src/pages/contact/contact.tsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React from "react";
+import './styles.css'
+
+export function Contact() {
+
+    return(<div className="contact">
+        <h1>Contact</h1>
+    </div>
+
```
Comment (ggiande) [inline src/pages/contact/contact.tsx]: Remove whitespace, please use PRETTIER plugin for formatting

File: src/pages/contact/styles.css [hunk 2]
```diff
@@ -0,0 +1,3 @@
+.contact {
+
```
Comment (ggiande) [inline src/pages/contact/styles.css]: Future code, remove or use it

File: src/pages/gallery/styles.css
Comment (ggiande) [inline src/pages/gallery/styles.css:1]: Remove empty file

File: src/pages/home/styles.css
Comment (ggiande) [inline src/pages/home/styles.css:1]: Expected for home to remove all stryling? If so, remove file

**Review by ggiande — CHANGES_REQUESTED** [review]
Needs some changes. After draft is complete, please promote to PR. Thanks

File: src/pages/contact/contact.tsx (hunk 1 above)
Comment (JonathanCardoza) [reply src/pages/contact/contact.tsx]: the page has been removed

File: README.md
Comment (JonathanCardoza) [reply README.md:1]: read me page has been added

**Review by ggiande — APPROVED** [review]
LGTM!

**Pull request merged**